apps/server/src/modules/board/board.module.ts

```diff
@@ -58,6 +58,10 @@ import { ColumnBoardCopyService } from './service/column-board-copy.service';
 		ContentElementService,
 		SubmissionItemService,
 		ColumnBoardCopyService,
+		/**
+		 * @deprecated - only used for deprecated lernraum module
+		 */
+		BoardNodeRepo,
```

Comment:
this is part of the hack. We need to export this repo, so that it can be used by learnraum